src/liquid/components/ld-button/ld-button.tsx

```diff
@@ -219,9 +219,7 @@ export class LdButton implements InnerFocusable, ClonesAttributes {
         {...this.clonedAttributes}
         aria-busy={hasProgress ? 'true' : undefined}
         aria-disabled={
-          this.disabled || this.el.getAttribute('aria-disabled') === 'true'
-            ? 'true'
-            : undefined
+          this.disabled || this.el.ariaDisabled ? 'true' : undefined
```

Comment:
```suggestion
          this.disabled || this.el.ariaDisabled === 'true' ? 'true' : undefined
```

`this.el.ariaDisabled` is a string and might also be 'false' or any other value (according to its type definition).